google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
Ah yes, this is why the solution works. I missed this bit of code, so feel free to ignore my question about why the solution works.